[Bug] [dolphinscheduler-api] create presto datasource failed with extra "jdbc connect parameters"

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

When try to create PRESTO datasource type, if jdbc connect parameters assigned.
TestConnect will be failed with  tips "JDBC connect failed".

For example, use `{"SSL":"true"}` as connect params:
![image](https://user-images.githubusercontent.com/2315481/187828629-e44af806-96a9-4923-8e0b-2de73e8d9d8f.png)

the logs will shows:
```
[ERROR] 2022-09-01 11:38:21.728 +0800 com.zaxxer.hikari.pool.HikariPool:[594] - HikariPool-2 - Exception during pool initialization.
java.sql.SQLException: Connection property 'SSL' is both in the URL and an argument
```

### What you expected to happen

Because "SSL" is a valid property when connect presto, so the test connect should be success.

### How to reproduce

As above.
Two points mentioned:
1. plz just use jdbc connet params at first to avoid the dataSourceClientCache, bcz cache key does not contain params.
2. IMO, when connect presto, we can set params either in jdbc-url or properties, However, the same parameter may not be specified using both methods, ref to [JDBC driver](https://trino.io/docs/current/client/jdbc.html#connection-parameters) 
I'm pleasure to fix this, and what is next?

### Anything else

Every time happens.

### Version

3.0.0

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-presto/src/main/java/org/apache/dolphinscheduler/plugin/datasource/presto/param/PrestoDataSourceProcessor.java
+++ b/dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-presto/src/main/java/org/apache/dolphinscheduler/plugin/datasource/presto/param/PrestoDataSourceProcessor.java
@@ -102,10 +102,11 @@ public class PrestoDataSourceProcessor extends AbstractDataSourceProcessor {
     @Override
     public String getJdbcUrl(ConnectionParam connectionParam) {
         PrestoConnectionParam prestoConnectionParam = (PrestoConnectionParam) connectionParam;
-        if (MapUtils.isNotEmpty(prestoConnectionParam.getOther())) {
-            return String.format("%s?%s", prestoConnectionParam.getJdbcUrl(),
-                    transformOther(prestoConnectionParam.getOther()));
-        }
+        /**
+         * return jdbc url without other parameters, avoid to assign same parameters both in URL and properties.
+         * or else got error msg: Connection property '...' is both in the URL and an argument
+         * ref to <a href="https://prestodb.io/docs/current/installation/jdbc.html">Presto JDBC Driver</>
+         */
         return prestoConnectionParam.getJdbcUrl();
     }
 

